android/app/src/main/kotlin/com/example/classroom_app/MainActivity.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.classroom_app
+
+import io.flutter.embedding.android.FlutterActivity
+
+class MainActivity: FlutterActivity()
```

Comment:
is this extending FlutterActivity()? how flutterActivity is being created?

Reply:
By extending FlutterActivity, MainActivity inherits functionality that allows it to run the Flutter code.  FlutterActivity automatically initializes the Flutter engine and runs the Flutter app in the Android environment.